packages/experimental/Menu/src/MenuPopover/MenuPopover.tsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
-import { stagedComponent } from '@fluentui-react-native/framework';
+import { stagedComponent, useFluentTheme } from '@fluentui-react-native/framework';
 import { Callout } from '@fluentui-react-native/callout';
 import { menuPopoverName, MenuPopoverProps } from './MenuPopover.types';
 import { useMenuPopover } from './useMenuPopover';
 
 export const MenuPopover = stagedComponent((props: MenuPopoverProps) => {
   const state = useMenuPopover(props);
+  const theme = useFluentTheme();
 
   return (_rest: MenuPopoverProps, children: React.ReactNode) => {
-    return <Callout target={state.triggerRef}>{children}</Callout>;
+    return (
+      <Callout target={state.triggerRef} borderWidth={1} borderColor={theme.colors.neutralStrokeAccessible}>
+        {children}
+      </Callout>
```

Comment:
Any reason this isn't a Slot with slotProps?

Reply:
laziness?

Actually I find it unlikely that someone would want to swap out the Callout with something else, especially since it handles the native popover logic/functionality, so I was wondering if it was worth making this a slot.

Reply:
I see. I've defaulted to just making everything a slot fit the sake of following patterns.

A real example of when you would want to swap out a Callout: One might want to replace it with just a View with absolute position (AKA, a floating view), for a platform that doesn't support Callout / more styling options / etc. I did that a lot as a "development shortcut" when Callout wasn't implemented on macOS, and will probably repeat the same for iOS.

 
Or maybe an inline menu 🤷 I think I saw an example of that in web.

Reply:
I see. I've defaulted to just making everything a slot fit the sake of following patterns.

A real example of when you would want to swap out a Callout: One might want to replace it with just a View with absolute position (AKA, a floating view), for a platform that doesn't support Callout / more styling options / etc. I did that a lot as a "development shortcut" when Callout wasn't implemented on macOS, and will probably repeat the same for iOS

 
Or maybe an inline menu 🤷 I think I saw an example of that in web.

Reply:
For inline menus the intent is that you just use MenuList, and leave out the Popover wrapping. So those cases should be fine.

Reply:
Yeah, re-reading this I realize they ship MenuList separately for the case of inline menus. So slots / this slots-as-jsx stuff both kinda do similar things I realize